syntax = "proto3";
package dna.node.proto;

message TrackFeatureProto {
    string node_id = 1;
    string track_id = 2;
    optional bytes bfeature = 3;
    string zone_relation = 4;
    int64 ts = 5;
};